code/modules/mob/living/simple_animal/bot/floorbot.dm

```diff
@@ -150,42 +157,42 @@
 	if(!..())
 		return
 
-	if(mode == BOT_REPAIRING)
+	if(mode == (BOT_REPAIRING || BOT_MAKE_TILE || BOT_EAT_TILE))
```

Comment:
This wont work, you'll need to check each individually
`mode == repairing || mode == make tile...`

Reply:
iirc I tested it and it worked fine, I will test again.

Reply:
Ok tested, it seems to work just fine?

Reply:
It wont, because you're comparing mode to integers, which doesnt work in this method. And if for some godforbidden reason it does, it does not comply with our guidelines